### ISSUE TITLE
be055438 - TabView has no transitions

### ISSUE THREAD

**Issue by AnonymHax** [papyros/qml-material#285]
Since the use of TabView Control (be055438)the transition from one tab to another is no longer animated

**Comment by iBeliever** [comment]
Yeah, unfortunately, to use the QtQuick.Controls TabView, we lost the animations. I'd like to try to find a way to add animations back in to match the Material Design spec.
